queries.sql

```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%'
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31'
+SELECT name FROM animals WHERE neutered = TRUE AND escape_attempts < 3
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu'
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5
+SELECT * FROM animals WHERE neutered = TRUE
+SELECT * FROM animals WHERE name != 'Gabumon'
+SELECT * FROM animals where weight_kg BETWEEN 10.4 AND 17.3 AND weight_kg BETWEEN 10.4 AND 17.3
+
```

Comment:
- [ ] Great job so far 👏🏻 however as requested by the previous reviewer you have to take a screenshot 📸 of your query results and add them to the pull request description.  This also helps to ensure that you run the queries with success. Kindly address this 👍🏻